moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -442,4 +449,36 @@ void MotionPlanningFrame::remoteUpdateGoalStateCallback(const std_msgs::EmptyCon
     }
   }
 }
+
+void MotionPlanningFrame::remoteUpdateStartStateRobotStateCallback(const moveit_msgs::RobotStateConstPtr& msg)
+{
+  moveit_msgs::RobotState msg_no_attached(*msg);
```

Comment:
This is the unchanged message right now and thus redundant.

I think it is a good idea though to restrict these start state changes to forbid changes in the attached collision objects.
We don't have that possibility in the RViz GUI either
and it might cause weird state effects in the GUI.

I would propose to add
```
msg_no_attached.attached_collision_objects.clear();
msg_no_attached.is_diff = true;
```
after the new variable definition here and below.